org-formation/organization.yaml

```diff
@@ -314,6 +315,7 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         Department: CNB
         Project: Challenges
+        CostCenter: NO PROGRAM / 000000
```

Comment:
This account is assigned with NO PROGRAM code because it is meant to be a "team account" where we host different challenges.  The idea is that the resources internal to this account should be tagged accordingly.